### DISCUSSION TITLE
ViVeTool 3.3 support

### DISCUSSION THREAD

**Discussion by boblinthewild** [PeterStrick/ViVeTool-GUI#53] [Q&A]
Hi -- I really like how easy it is to use the GUI. A new version of ViVeTool was just released, and I'm wondering if the GUI will work with it? Thanks!

**Comment by PeterStrick** [comment]
Currently the GUI is using an old Version of the ViVeTool API that doesn't have support for Feature Variants. A new ViVeTool GUI Version 1.7 is in the works and half finished in the `v1.7-Testing` Branch, I just currently don't really have time to work on it due to school, work and friends

**Reply by boblinthewild** [reply]
That's perfectly understandable. Thanks for the quick reply!
